src/sidebar/components/sidebar-content.js

```diff
@@ -365,6 +389,8 @@ function SidebarContentController(
 
     store.clearSelectedAnnotations();
     store.selectTab(selectedTab);
+    // Clear direct-linked group fetch failed state.
```

Comment:
I just realized there's no test for this so I'll have to add that tomorrow. In fact there is not test for this method at all in the tests currently. These tests desperately need some refactoring but I didn't want to do all that here.

Reply:
Is this resolved or still TODO?

Reply:
Just added it a moment ago.